client/src/views/Home/JoinParty.jsx

```diff
@@ -13,6 +17,11 @@ function JoinGame({
 }) {
   const [roomCode, setRoomCode] = useState(slug);
   const [isJoiningParty, setIsJoiningParty] = useState(false);
+  const [howToPlayOpen, setHowToPlayOpen] = useState(false);
+
+  // Show and hide content about how to play with charades app
```

Comment:
please remove this comment. The functions are properly named and very easy to understand as it is a common pattern (and very small).